fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -190,6 +191,14 @@ private void processDropColumn(DropColumnClause alterClause, OlapTable olapTable
             if (isKey) {
                 throw new DdlException("Can not drop key column in primary data model table");
             }
+            MaterializedIndexMeta indexMeta = olapTable.getIndexMetaByIndexId(olapTable.getBaseIndexId());
```

Comment:
1. How about modifying the type of sort columns?
2. I want to separate the sort column and primary key column on existing tables. How to handle it?
    That may come from a scenario. The original table is not designed. After we release the feature, they want to use the feature but must reload the data. Can we simplify the logic by providing a `alter table` command?

Reply:
yes, it is possible, can do it in next PR.